src/Microsoft.Framework.OptionsModel/project.json

```diff
@@ -6,6 +6,10 @@
     "Microsoft.Framework.DependencyInjection.Abstractions": "1.0.0-*",
     "Microsoft.Framework.NotNullAttribute.Sources": { "type": "build", "version": "1.0.0-*" }
   },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/aspnet/options"
```

Comment:
So does this work out of the box? I read https://github.com/aspnet/dnx/issues/1901 but I don't see how this info here is enough for it to completely work.